src/TurbulenceClosures/closure_tuples.jl

```diff
@@ -54,6 +54,25 @@ for (f, alt_f) in zip(funcs, alt_funcs)
     end
 end
 
+
+#####
+##### Sum of diffusive and viscous fluxes, and other relevant functions
+#####
+
+for diff_flux in (:diffusive_flux_x, :diffusive_flux_y, :diffusive_flux_z,
+                  :viscous_flux_ux, :viscous_flux_uy, :viscous_flux_uz,
+                  :viscous_flux_vx, :viscous_flux_vy, :viscous_flux_vz,
+                  :viscous_flux_wx, :viscous_flux_wy, :viscous_flux_wz,
+                  funcs..., alt_funcs...)
+    # Unroll the loop over a tuple
+    @eval @inline $diff_flux(i, j, k, grid, closure_tuple::Tuple, diffusivity_fields, args...) =
+        $diff_flux(i, j, k, grid, closure_tuple[1], diffusivity_fields[1], args...) +
+        $diff_flux(i, j, k, grid, closure_tuple[2:end], diffusivity_fields[2:end], args...)
+
+    # End of the line
+    @eval @inline $diff_flux(i, j, k, grid, closure_tuple::Tuple{}, args...) = zero(grid)
+end
```

Comment:
can you implement these by adding them to the list of functions that we already unroll over tuples?

Reply:
Do you mean adding them to this bit? https://github.com/CliMA/Oceananigans.jl/blob/200f0622278fcfa583da0119e9696048b334146f/src/TurbulenceClosures/closure_tuples.jl#L21-L55

Reply:
Yes. Probably best to build up the master list from a few components.

Reply:
Okay, sure. A quick question though: is there a reason why this is hard-coded up to 5 elements in the tuple?